### ISSUE TITLE
link to existing resources

### ISSUE THREAD

**Issue by afeld** [GSA/it-accessibility-playbook#1]
Just saw this repository get created...especially while it's in development, would be great to have the README link to relevant resources, e.g. https://pages.18f.gov/accessibility/.

/cc @nickbristow

**Comment by lachellel** [comment]
Yep!  The accessibility council has a handbook that is currently a word document.  Our colleagues have been developing a digital version in their own repos.  I opened this GSA repo to accept the pull requests for the transition over from personal to govt. 

The intent is to deploy under *.cio.gov, and also link into digital services playbook, the 18f accessibility (more detailed implementation guides), etc.   

[The current README is a placeholder only ... stay tuned!]
